sdks/python/apache_beam/io/bigquery.py

```diff
@@ -374,7 +374,7 @@ def __init__(self, table=None, dataset=None, project=None, query=None,
       self.use_legacy_sql = True
     else:
       self.query = query
-      self.use_legacy_sql = use_legacy_sql
+      self.use_legacy_sql = not use_standard_sql
```

Comment:
create a JIRA issue to fix the internals later. It would make sense to use `use_standard_sql` internally as well.